cmd/state-svc/main.go

```diff
@@ -38,26 +38,40 @@ const (
 func main() {
 	var exitCode int
 
+	var cfg *config.Instance
 	defer func() {
 		if panics.HandlePanics(recover(), debug.Stack()) {
 			exitCode = 1
 		}
+
+		if err := cfg.Close(); err != nil {
+			logging.Error("Failed to close config after exiting systray: %w", err)
+		}
+
 		if err := events.WaitForEvents(5*time.Second, rollbar.Wait, rollbar.Close, authentication.LegacyClose, logging.Close); err != nil {
 			logging.Warning("Failing to wait for rollbar to close")
```

Comment:
Just noticed this says "rollbar" rather than "events". Simple tweak we can probably include in this PR.